client/components/Events/index.js

```diff
@@ -1,19 +1,24 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { Link } from 'react-router';
+import React, { Component } from "react";
```

Comment:
I'd prefer the use of `'` single quotes for JS and `"` for JSX cos of the ease of typing a single quote in JS.

JSX and HTML are similar and it makes more sense to write HTML values in double quote than double quotes.

Just my thoughts.

Reply:
The default tabWidth indent is actually 2 not 4 also I'll switch to single quotes instead.